portfolio.js

```diff
@@ -13,15 +15,26 @@ Project.prototype.toHtml = function() {
   $newProject.find('a').text(this.title);
```

Comment:
You are cloning the wrong thing when constructing the filled-in HTML for your projects. Instead of cloning the entire projects section, you just want to clone the .template article.